pages/job-category/index.js

```diff
@@ -64,15 +64,17 @@ const StyledSection = styled.section`
   width: 80 %;
 `;
 
-const StyledSpan1 = styled.span``;
+const StyledButton = styled.button``;
+
+const StyledLink = styled(Link)``;
 
 export default function createJobcategory() {
   return (
     <StyledDiv>
       <StyledDiv1>
```

Comment:
When you stick to a naming convention such as Styled[HTML-TAG] then you run into this issue sooner or later. `StyledDiv1`, `StyledDiv2` ... That's not good. You should name your components according to what they do.

Reply:
I think we still want to resolve the naming issues here. 

I can still see `StyledDiv` and `StyledDiv`. 

We would ideally want more meaningful naming for these. Potentially something link `StyledCard` or `Card` seeing as that was the old class name you had for them.